cobalt/cobalt.cc

```diff
@@ -28,22 +28,22 @@ int main(int argc, const char** argv) {
 
   // TODO: (cobalt b/375241103) Reimplement this in a clean way.
 
-  constexpr auto cobalt_args = std::to_array<const char*>({
-    // Disable first run experience, kiosk, etc.
-    "--disable-fre", "--no-first-run", "--kiosk",
-    // Enable Blink to work in overlay video mode
-    "--force-video-overlays",
-    // Disable multiprocess mode.
-    "--single-process",
-    // TODO(mcasas): Add "--ozone-platform=starboard".
-    // Enable remote Devtools access.
-    "--remote-debugging-port=9222",
-    "--remote-allow-origins=http://localhost:9222",
-    // This flag is added specifically for m114 and should be removed after
-    // rebasing to m120+
-    "--user-level-memory-pressure-signal-params",
-    "https://www.youtube.com/tv"
-  });
+  constexpr auto cobalt_args = std::to_array<const char*>(
```

Comment:
Playing around with this, the current `enable_vulkan=false` is not enough to disallow vulkan usage in Cobalt builds; a `--disable-features=Vulkan` needs to be added to the command line here (this is probably an oversight upstream).